src/styles/main.scss

```
@@ -1,3 +1,60 @@
+$square-count: 31;
+$square-size: 100px;
+// $days: mon, tue, wed, thu, fri, sat, sun;
+
 body {
   margin: 0;
 }
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: 710px;
+  gap: 1px;
+  // margin: 0 auto;
+}
+
+.calendar__day {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: $square-size;
+  height: $square-size;
+  box-sizing: border-box;
+  background-color: #eee;
+  border: 1px black solid;
+
+  @for $index from 1 through $square-count {
+    &--#{$index}::before {
+      content: "#{$index}";
+    }
+  }
+
```

```suggestion
@for $date from 28 through $month-length {
    &--month-length--#{$date} :nth-child(n + #{$date + 1}) {
      display: none;
    }
  }
```
also add this block of code